python/setup.py

```diff
@@ -530,7 +540,8 @@ def has_ext_modules(foo):
 install_requires = (
     'numpy >= 1.14',
     'six >= 1.0.0',
-    'futures; python_version < "3.2"'
+    'futures; python_version < "3.2"',
+    'enum34 >= 1.1.6; python_version < "3.4"',
```

Comment:
You must also update `requirements.txt`.

Reply:
Done!